README.md

```diff
@@ -1,32 +1,44 @@
-# Asylum RG BE
+# Asylum Report Generator Backend
+
+A **search tool** to give a user-friendly view into a large dataset of asylum case decisions. <br/>
+This concept is similar to the [TRAC database](https://trac.syr.edu/phptools/immigration/asylum/) — a popular 
+existing tool, but with a far more robust ability to select and filter data.
+* A user can generate a table of results, filtered by multiple parameters
+* A user can generate a real-time graph to represent what trend looks
+  like over time (Release 4)
+* A user can generate a heatmap of asylum case outcomes (Release 5)
+* A user can download the results of their search (Release 6)
+
+### Design Document / Repos / Setup
+Document can be found [**HERE**](documentation/design_document.md) <br/>
+Docs repo can be found [**HERE**](https://github.com/BloomTech-Labs/asylum-rg-docs) <br/>
```

Comment:
May, I suggest to link like this? Totally up to you. Just a suggestion.

[Product Document Repository ](https://github.com/BloomTech-Labs/asylum-rg-docs)
[Frontend Repository](https://github.com/BloomTech-Labs/asylum-rg-fe)